WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -200,6 +208,7 @@ private async Task<ConnectionConfirmationResponse> ConfirmConnectionCoreAsync(Co
 			{
 				throw new WabiSabiProtocolException(WabiSabiProtocolErrorCode.IncorrectRequestedAmountCredentials, $"Round ({request.RoundId}): Incorrect requested amount credentials.");
 			}
+			RequestTimeStatista.Instance.Add("connection-confirmation-inside1", DateTimeOffset.UtcNow - beforeInside);
```

Comment:
Why split this into `connection-confirmation-inside1` and `connection-confirmation-inside2`?

Reply:
Hmm, good question. I just added logging into every lock and here are two locks.

Reply:
I am interested in the run time without the lock. So we can be sure about if the lock takes away the time or the internal calculations. We are almost sure it is the lock - but I want to have confirmation about that.